<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BruteForcingLog</class>
 <widget class="QDialog" name="BruteForcingLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="bfl_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP Camera Brute Forcing Log:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="bfl_listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="back_pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
